cypress/support/commands.d.ts

```diff
@@ -1,20 +1,62 @@
 declare namespace Cypress {
   interface Chainable {
     /**
-     * Selects a Toastify element by its content
-     * @example cy.selectToastByContent('content')
+     * Target a page to test elements within it
+     * @example cy.targetPage('title')
      */
-    selectToastByContent(content: string);
+    targetPage(noteTitle: string): Chainable<JQuery<HTMLElement>>;
+
+    /**
+     * Assert there is a certain amount of backlinks
+     * @example cy.numberOfReferencesShouldEqual('linked', 5)
+     */
+    numberOfReferencesShouldEqual(amount: number, referenceType: string);
```

Comment:
Similar to another comment, I think this could be easier to read/use if you split this up into `numberOfLinkedReferencesShouldEqual` and `numberOfUnlinkedReferencesShouldEqual`